apps/skolplattformen-sthlm/android/app/src/debug/java/com/app/UnsecureCookieJar.java

```diff
@@ -0,0 +1,36 @@
+package com.app;
+
+import android.util.Log;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import okhttp3.Cookie;
+import okhttp3.CookieJar;
+import okhttp3.HttpUrl;
+
+public class UnsecureCookieJar implements CookieJar {
```

Comment:
Oh, this isn't being used. I initially added my own cookie jar, but moved to using the existing one instead and just feeding it all set cookie headers.